kickstart/etc/nginx-posm.conf

```diff
@@ -34,13 +36,18 @@ server {
   location /fp-tiler {
     proxy_pass http://127.0.0.1:{{fp_tiler_port}};
   }
-  location /fp-tasks {
-    proxy_pass http://127.0.0.1:{{fp_tasks_port}};
+  location /fp-tasks/ {
+    proxy_pass http://127.0.0.1:{{fp_tasks_port}}/;
```

Comment:
(We couldn't use this approach for fp-tiler because it was using the provided path to generate output, but for cases where that's not needed, this is good for stripping prefixes.)